tortoise/backends/asyncpg/client.py

```diff
@@ -42,85 +42,61 @@ def __init__(self, user: str, password: str, database: str, host: str, port: Sup
         self.host = host
         self.port = int(port)  # make sure port is int type
 
-        self.dsn = self.DSN_TEMPLATE.format(
-            user=self.user,
-            password=self.password,
-            host=self.host,
-            port=self.port,
-            database=self.database
-        )
-        self._db_pool = None  # Type: Optional[asyncpg.pool.Pool]
+        # self._db_pool = None  # Type: Optional[asyncpg.pool.Pool]
```

Comment:
I think we don't need commented code in distributed lib, it would just confuse people if they see it

Reply:
:man_facepalming: